pom.xml

```diff
@@ -84,6 +84,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
I can't find information on this exact version, but isn't that the deprecated 'warnings' plugin? Is it possible to use [warnings-ng](https://plugins.jenkins.io/warnings-ng) instead?

Reply:
Yes, but in a limited way. The parser could not be created by the library on the fly but needs to be present in the Jenkins config.

Reply:
[JENKINS-56055](https://issues.jenkins-ci.org/browse/JENKINS-56055?jql=creator%20in%20(currentUser()))

Reply:
switched to `warning-ng` plugin